compiler/stable_mir/src/compiler_interface.rs

```diff
@@ -121,6 +121,9 @@ pub trait Context {
     /// Get the instance type with generic substitutions applied and lifetimes erased.
     fn instance_ty(&self, instance: InstanceDef) -> Ty;
 
+    /// Get the instance signature with .
+    fn instance_sig(&self, def: InstanceDef) -> FnSig;
```

Comment:
```suggestion
    fn instance_sig(&self, def: InstanceDef) -> FnSig;
```

Function signatures are necessarily late-bound. I don't think we should be returning `FnSig` here -- if you need to erase the late-bound regions at a call-site, please add a helper function to `Binder`.

Reply:
That makes sense. I think what we really need is to create an API to get the function ABI. That's more helpful at later stages than the FnSig.

Reply:
That should be a function from `Instance` -> `FnAbi`. See `fn_abi_of_instance` in the compiler.

Reply:
Yeah, I just need to create the stable structures. I'll create a follow up PR with that. Do you think it's worth having both, or should I just remove this one?

Reply:
Just remove this one unless you need it otherwise. I consider it easier to add APIs when it becomes compelling to add them.